prism/src/main/java/org/apache/falcon/resource/AbstractEntityManager.java

```diff
@@ -172,6 +173,9 @@ protected static void checkColo(String colo) {
             for (String cluster : clusters) {
                 try{
                     Cluster clusterEntity = EntityUtil.getEntity(EntityType.CLUSTER, cluster);
+                    if(type.equals(EntityType.PROCESS)){
+                        EntityUtil.getEntity(EntityType.PROCESS,entity.getName());
```

Comment:
What does this code achieve? I do not understand how this will fix the issue, please elaborate.